### PR TITLE
Add client version convention

### PR DESCRIPTION
**Pull request by danielgomezrico**
Should we send the same header on each app or a new header based on app name?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # code_style
 Code styles, and tools for different languages
+
+# General conventions
+
+- Mobile Clients will send a header `bv-app-version` with the app version for each api request.
```
Comment (orendon) [inline README.md]: +1 for `"bv-app-version" "platform/version"`
with `platform` being either `android` or `ios` and `version` following semver standars

Comment (danielgomezrico) [reply README.md]: +1 @orendon idea

Comment (edwaraco) [reply README.md]: ¿What happens if we have one or more mobile application for each platform. For example an application for customer and another application for admin?
I believe the `bv-app-version` should contain `platform/typeApp/version`; 
`platform`: maybe `android` or `iOS`
`typeApp`: i.e `Customer` or `admin`
`version`: following semver standars (`x.y.z`)

**Comment by danielgomezrico** [comment]
@earayo +1 idea!

File: README.md (hunk 1 above)
Comment (danielgomezrico) [reply README.md]: @earayo +1 edward idea `platform/typeApp/version`

Comment (Busta117) [reply README.md]: we can add "build" at the end... so,  `platform/typeApp/version/build`

Comment (Busta117) [reply README.md]: so.. `platform/typeApp/version/build` where:
`platform` : platform who make the request in **lower case** (ej: `ios`, `android`)
`typeApp` :  app profile who make the request in **lower case** (ej: `customer`, `driver`, `admin`) if there are only one app could be `mobile` for mobile apps
`version` : current app version following semver standard (`x.y.z`)
`build` : current build version, could be empty, number or letters 

am i right?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+# General conventions
+
+- Mobile Clients should always send the next headers for every request with our api: 
+
+  * `bv-app-version`: `platform/typeApp/version/build` _where_:
+  
+    - `platform` : platform who make the request in lower case (ej: ios, android)
+    - `typeApp` : app profile who make the request in lower case (ej: customer, driver, admin) if there are only one app could be mobile for mobile apps
+    - `version` : current app version following semver standard (x.y.z)
+    - `build : current build version, could be empty, number or letters
```
Comment (lordaniel) [inline README.md]: @caipivara Is missing an end ` for build ?

Comment (orendon) [reply README.md]: Good catch

**Review by orendon — APPROVED** [review]
Besides the lord comment :shipit:

**Review by lordaniel — APPROVED** [review]
@caipivara Good Job

File: README.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
+# General conventions
+
+- Mobile Clients should always send the next headers for every request with our api: 
+
+  * `bv-app-version`: `platform/typeApp/version/build` _where_:
+  
+    - `platform` : platform who make the request in lower case (ej: ios, android)
+    - `typeApp` : app profile who make the request in lower case (ej: customer, driver, admin) if there are only one app could be mobile for mobile apps
+    - `version` : current app version following semver standard (x.y.z)
+    - `build` : current build version, could be empty, number or letters
```
Comment (edwaraco) [inline README.md]: I don't like the `build` could be empty. What do you think about to avoid it?

**Review by edwaraco — CHANGES_REQUESTED** [review]
@caipivara Good job!  I add a comment for your consideration.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,13 @@
 # code_style
 Code styles, and tools for different languages
+
+# General conventions
+
+- Clients should send the next headers on every request to our api:
+
+  * `bv-app-version`: `platform/typeApp/version/build` (***each value separated by a `/`***) _where_:
+
+    - `platform` :***mandatory*** platform who make the request in lower case (ej: ios, android)
```
Comment (edwaraco) [inline README.md]: What do you think:
`platform` (***mandatory***): ...

File: README.md [hunk 5]
```diff
@@ -1,2 +1,13 @@
...
+    - `typeApp` :***mandatory*** app profile who make the request in lower case (ej: customer, driver, admin) if there are only one app could be mobile for mobile apps
```
Comment (edwaraco) [inline README.md]: Same here.
`typeApp` (***mandatory***): ...

File: README.md [hunk 6]
```diff
@@ -1,2 +1,13 @@
...
+    - `version` :***mandatory*** current app version following semver standard (x.y.z)
```
Comment (edwaraco) [inline README.md]: Same here:
`version` (***mandatory***): ...

File: README.md [hunk 7]
```diff
@@ -1,2 +1,13 @@
...
+    - `build` :***optional*** current build version, could be number or letters
```
Comment (edwaraco) [inline README.md]: Wdyt?
`build` (***optional***): ...

**Review by edwaraco — CHANGES_REQUESTED** [review]
Good job. I added some more silly comments.

**Review by edwaraco — APPROVED** [review]
@Busta117 and @caipivara good job. I'm sorry for my dyslexia.

**Pull request merged**